src/robot/utils/robottypes.py

```diff
@@ -57,3 +57,7 @@ def is_truthy(item):
 def is_falsy(item):
     """Opposite of :func:`is_truthy`."""
     return not is_truthy(item)
+
+
+def is_subscriptable(item):
+    return hasattr(item, '__getitem__')
```

Comment:
I'm not certain is this helper needed or should we just inline it. Perhaps it will be useful in the future so leaving it makes sense.